libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/usePanelTabs.tsx

```diff
@@ -166,57 +175,75 @@ export const useCreateWorkflowPanelTabs = ({
             : Constants.TEMPLATE_PANEL_TAB_NAMES.REVIEW_AND_CREATE,
         hasError: Object.values(workflows).some((workflowData) => workflowData.errors.kind || workflowData.errors.workflow),
         isCreating,
-      }),
-    }),
-    [intl, dispatch, isMultiWorkflowTemplate, connectionsExist, parametersExist, workflows, isCreating]
-  );
+      },
+      onClosePanel
+    );
+    tab.footerContent.secondaryButtonDisabled = !showCloseButton;
```

Comment:
I think this is a bit confusing for the author who write basicsTab method above. This property is overriden after the author returned this property causing the behavior is not what the author expected. Also this line assumed that the author uses the second button for the close button. I would suggest to pass showCloseButton to be passed to the basicsTab method and handled over there.